citadel/install_windows.md

```diff
@@ -78,4 +78,4 @@ conda uninstall libignition-<package_name> --channel conda-forge
 
 ## Troubleshooting
 
-See [Troubleshooting](troubleshooting.md#windows)
+See [Troubleshooting](/docs/citadel/troubleshooting#windows)
```

Comment:
Looks like the [Citadel troubleshooting page](https://ignitionrobotics.org/docs/citadel/troubleshooting#windows) actually doesn't have a Windows section. The link will still go to the page, it just won't jump to a windows section. I don't know if we want to keep this in case we add a Windows section in the future, or just remove it so that the link is correct. I'm leaning toward the latter, just removing `#windows`. Mind doing that?